index.html

```diff
@@ -500,8 +500,8 @@
         <dfn><a href=
         "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
         <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>,
-        and the following exception names are defined in [[!WEBIDL-1]]:
+        "https://heycam.github.io/webidl/#ArrayBufferView">ArrayBufferView</a></dfn>,
+        and the following exception names are defined in [[!WEBIDL-2]]:
```

Comment:
I don't think this phrase is required anymore since the entire spec is using WEBIDL-2?